bitnami/tomcat/templates/_pod.tpl

```diff
@@ -178,6 +178,16 @@ volumes:
     emptyDir: {}
   {{- end }}
   {{- end }}
+  {{- if and .Values.metrics.jmx.enabled .Values.metrics.jmx.config (not .Values.metrics.jmx.existingConfigmap) }}
+  - configMap:
+      name: {{ template "tomcat.fullname" . }}-jmx-configuration
+    name: jmx-config
+  {{- end }}
+  {{- if and .Values.metrics.jmx.enabled .Values.metrics.jmx.existingConfigmap }}
+  - configMap:
+      name: {{ .Values.metrics.jmx.existingConfigmap | quote }}
+    name: jmx-config
+  {{- end }}
```

Comment:
Could you please use the `tomcat.metrics.jmx.configmapName` helper instead?

```suggestion
  {{- if and .Values.metrics.jmx.enabled (or Values.metrics.jmx.config .Values.metrics.jmx.existingConfigmap) }}
  - configMap:
      name: {{ include "tomcat.metrics.jmx.configmapName" . }}
    name: jmx-config
  {{- end }}

Reply:
Done.